servers/account-data-deleter/src/dataService/exportStateService.ts

```diff
@@ -48,15 +48,26 @@ export class ExportStateService {
       body: result,
     });
     if (result != null && ExportStateService.isComplete(result)) {
-      const signedUrl = await this.getExportUrl(
-        payload.detail.prefix,
-        payload.detail.encodedId,
-      );
-      await this.notifyUser(
-        payload.detail.encodedId,
-        payload.detail.requestId,
-        signedUrl,
-      );
+      try {
+        const signedUrl = await this.getExportUrl(
+          payload.detail.prefix,
+          payload.detail.encodedId,
+        );
+        await this.notifyUser(
+          payload.detail.encodedId,
+          payload.detail.requestId,
+          signedUrl,
+        );
+      } catch(error) {
+        Sentry.addBreadcrumb({
+          message: 'ExportStateService - processUpdate failed',
+          data: {
+            requestId: payload.detail.requestId,
+            encodedId: payload.detail.encodedId,
+          },
+        });
+        Sentry.captureException(error);
```

Comment:
Do we want to re-throw the error here as well?
```suggestion
        Sentry.captureException(error);
        throw error;
```

Reply:
good catch!